src/project.test.ts

```diff
@@ -66,31 +66,68 @@ describe('project', () => {
           rootPackage,
           workspacePackages,
           isMonorepo: true,
+          releaseInfo: {
+            releaseDate: new Date(2022, 6, 22),
+            releaseNumber: 1234,
+          },
         });
       });
     });
 
-    it('collects information about a polyrepo project', async () => {
```

Comment:
I decided to drop the polyrepo test from this because `readProject` doesn't technically support polyrepo projects right now and that is more true with this PR than before. I think this is something we can fix in a later PR.